src/code/actors/morsetx.js

```diff
@@ -61,7 +65,7 @@ class Morse extends Actor {
 }
 
 function morseFactory(game, group, door, player = 1) {
-  let transmissions = getSignal(door.difficulty)
+  let transmissions = door ? getSignal(door.difficulty) : getSignal()
```

Comment:
This may not work as expected. If the door type is `0`, then the ternary operator will call `getSignal()`, effectively randomizing the result.

Reply:
Nevermind, I just realized door is an object and not a value.